tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -212,3 +212,31 @@ Feature: antivirus
       | eicar.com     | virusFile1.txt |
       | eicar_com.zip | virusFile2.zip |
 
+  @env-config
+  Scenario Outline: upload a file with virus smaller than the upload threshold
+    Given the config "ANTIVIRUS_MAX_SCAN_SIZE" has been set to "100"
+    And using <dav-path-version> DAV path
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicar.com" to "/aFileWithVirus.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Alice" should get a notification with subject "Virus found" and message:
+      | message                                                                             |
+      | Virus found in aFileWithVirus.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Alice" file "/aFileWithVirus.txt" should not exist
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+  @env-config
+  Scenario Outline: upload a file with virus larger than the upload threshold
+    Given the config "ANTIVIRUS_MAX_SCAN_SIZE" has been set to "100"
+    And using <dav-path-version> DAV path
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicarcom.zip" to "/aFileWithVirus.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
```

Comment:
Any notification that use gets after upload?

Reply:
If the virus file size is greater than the threshold, it doesn't scan the file and thus no notification is shown just for the virus upload.